__tests__/Utilities.test.js

```diff
@@ -110,6 +112,58 @@ describe('Utilities', () => {
 
   })
 
+  describe('generateTitle()', () => {
```

Comment:
i'm finding "generateTitle()" to be a confusing name.  Are we talking about the popover title?  would popoverTitle or getPopoverTitle() be a better name?  It makes me wonder if the test I commented on about the word title in https://github.com/LD4P/sinopia_editor/pull/567  is about the popover title ...

Reply:
Maybe the refactoring @jgreben 	is now exploring will present the opportunity rename/rework this as well.

Reply:
being removed